templates/open-tech-will-save-us.html

```diff
@@ -27,10 +28,10 @@ <h2>Previous Events</h2>
             {% if loop.first %}
             {% else %}
                 <div class="otwsu-episode">
-                    <img src="/open-tech-will-save-us/otwsu{{episode.code}}.png"
+                    <img src="/open-tech-will-save-us/otwsu{{ number_of_episodes + 1 - loop.index }}.png"
```

Comment:
I think it would be nice to move this into a variable since it's used twice, and the lines are somewhat long with it.